### ISSUE TITLE
Cron schedule regression supporting intervals

### ISSUE THREAD

**Issue by chrisleavoy** [reef-pi/reef-pi#1978]
In earlier versions of reef-pi, we could specify doser schedules such as `*/15` that would run every 15 minutes. I noticed in version 5.2 that I cannot edit my existing doser schedule because the UI says its invalid: `An integer, a range (e.g. 1-5) or * is required!`

<img width="1716" alt="image" src="https://user-images.githubusercontent.com/1428245/204152229-759fbecc-5f64-44d6-80dd-3266f59d518a.png">

It'd be great if the form input validation could be relaxed so this functionality is restored.  Cheers!

**Comment by ranjib** [comment]
Yes , and i removed that validation in 5.3 . Is I’m the ui still showing
validation error with 5.3 or 6 ?
